### PR TITLE
Fix #2515: Updated literals to named arguments

### LINKED ISSUE #2515: Update literals to named arguments in TopicLessonsFragmentTest.kt
**Is your feature request related to a problem? Please describe.**
Update the literals passed as argument to named arguments. This improves code readability and provides better context to the readers. One such instance is shown as an example below.

**Describe the solution you'd like**
Update the argument in line 246 below:
https://github.com/oppia/oppia-android/blob/bb7351b4c0700b3dadd1a51edd99e8e51c696eb4/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt#L244-L248

to the following code:
```        
        atPositionOnView(
          R.id.story_summary_recycler_view,
          position = 1,
          R.id.chapter_recycler_view
        )
```

**Additional context**
1. Identify other instances of such changes in `TopicLessonsFragmentTest`, update the code with named arguments. 
2. Identify other files/classes where such changes, update with named arguments for better readability.

### PR DESCRIPTION
**Pull request by harshpatel63**
## Explanation

Fixes #2515. Updated literals passed as arguments to named arguments for better readability.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
Suggesting to follow point 3 from the installation step.
https://github.com/oppia/oppia-android/wiki#installation

**Review by FareesHussain — CHANGES_REQUESTED** [review]
Update PR title and add an explanation in PR description as mentioned in the checklist

update PR title to `Fix #2515: Updated literals to named arguments`
and add `Fixes #2515` in the Explanation in PR description

**Review by FareesHussain — CHANGES_REQUESTED** [review]
There are a lot of line that exceeds max line limit (i.e 100) make sure to fit all of them within 100 lines

**Comment by harshpatel63** [comment]
> There are a lot of line that exceeds max line limit (i.e 100) make sure to fit all of them within 100 lines

Should I divide the total 600 lines into 6 parts and then fix these in parts.

**Comment by FareesHussain** [comment]
> > There are a lot of line that exceeds max line limit (i.e 100) make sure to fit all of them within 100 lines
> 
> Should I divide the total 600 lines into 6 parts and then fix these in parts.

No, You can do it in the same PR, its just that you need to fix the lines mentioned [here](https://github.com/oppia/oppia-android/pull/2523/checks?check_run_id=1733945816) by limiting them to 100 characters (make sure the characters do not exceed the vertical line)

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -5,6 +5,22 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```
Comment (srikanthkb) [inline .idea/codeStyles/Project.xml:8]: These files are auto-generated by IntelliJ. They don't have to be included in your PR.
Check https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR for more info on the guidelines. Also, please self-review once, before checking the checklist while creating a PR.

**Comment by srikanthkb** [comment]
@Sci3fic 
I think it will be hard to review 300+ files in this PR. I suggest you make changes in `TopicLessonsFragmentTest` first, and check for instances where the change is restricted to argument `position` in other **test** files. 
This will also make it easier to understand the workflow for Oppia-Android.

**Pull request closed without merging**